### PR TITLE
Adding 1 site

### PR DESCRIPTION
**Pull request by C3n7ral051nt4g3ncy**
## Added `1 site`: 

- `counter_social`

### CODE REVIEW

**Comment by C3n7ral051nt4g3ncy** [comment]
## Added `2nd site`:

- `sofurry`

File: wmn-data.json [hunk 1]
```diff
@@ -1240,6 +1240,18 @@
         "cat" : "art",
         "valid" : true
        },        
+       {
+        "name" : "counter_social",
+        "uri_check" : "https://counter.social/@{account}",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : "@counter.social) - CounterSocial</title>",
+        "m_string" : "The page you are looking for isn't here.",
```
Comment (WebBreacher) [inline wmn-data.json:1249]: Can you please verify the `m_code`? In Chrome I see the `'` is `&39;` so the `m_code` value should be `The page you are looking for isn&#39;t here.`

**Comment by C3n7ral051nt4g3ncy** [comment]
## Added `3rd site`:

- `myportfolio`

**Comment by C3n7ral051nt4g3ncy** [comment]
## Added `4th site`: 

- `Paypal`

File: wmn-data.json [hunk 2]
```diff
@@ -3629,6 +3641,18 @@
...
         "known" : ["Djelizamay","Andysparkles"],
         "cat" : "social",
+        "valid" : true
+       },        
+       {
+        "name" : "myportfolio",
+        "uri_check" : "https://{account}.myportfolio.com",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : "html lang",
```
Comment (WebBreacher) [inline wmn-data.json:3651]: Please choose a more distinct string here. Some of our targets and users may not be english readers and this value could be dynamic. Perhaps something like `preserve-whitespace">Back to Top` and add `/work` to the end of the `uri_check`?

File: wmn-data.json [hunk 3]
```diff
@@ -4004,6 +4028,18 @@
...
         "cat" : "finance",
+        "valid" : true
+        },        
+        {
+        "name" : "Paypal",
+        "uri_check" : "https://www.paypal.com/paypalme/{account}",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : "userInfo",
+        "m_string" : "Get your very own PayPal.Me link",
```
Comment (WebBreacher) [inline wmn-data.json:4039]: Let's try to make the `m_string` a component of the page not text shown in the browser as people that use languages other than English will get false positives.

File: wmn-data.json [hunk 4]
```diff
@@ -3629,6 +3641,18 @@
...
         "known" : ["Djelizamay","Andysparkles"],
         "cat" : "social",
+        "valid" : true
+       },        
+       {
+        "name" : "myportfolio",
+        "uri_check" : "https://{account}.myportfolio.com/work",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : "digital artist, illustrator, game art",
```
Comment (WebBreacher) [inline wmn-data.json:3651]: This is something that varies based upon the user account. For instance, https://bqwphotography.myportfolio.com does not have those data points.

Comment (WebBreacher) [reply wmn-data.json:3651]: How about `class="page-title"`

**Pull request merged**